pkg/installer/utils.go

```diff
@@ -214,3 +218,102 @@ func pullManifest(url string) (string, error) {
 	}
 	return string(contents), nil
 }
+
+// storageClassToManifest returns a manifest for storageClass
+func storageClassToManifest(storageClass *kstoragev1.StorageClass) ([]byte, error) {
+	tmp := &kstoragev1.StorageClass{}
```

Comment:
Why we need to highlight with the name that this is a temporary SC?

Reply:
I guess it's temporary in that it is just used to store the data before we convert to manifest, would `newStorageClass` be better?

Reply:
yepp, thanks